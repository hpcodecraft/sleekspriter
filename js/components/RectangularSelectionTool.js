var RectangularSelectionTool = React.createClass({
  render: function() {
    return (
      <div id="RectangularSelection-Tool" className="ToolComponent">
        <i className="icon flaticon-selection7"></i>

        <span className="hint"></span>
      </div>
    );
  }
});