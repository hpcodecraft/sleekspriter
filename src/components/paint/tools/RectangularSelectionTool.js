import React from "react";
import { t } from "../../../utils";

const RectangularSelectionTool = () => {
  return (
    <div id="RectangularSelection-Tool" className="ToolComponent">
      <i className="icon flaticon-selection7" />
      <span className="hint">{t("Select some pixels to work with!")}</span>
    </div>
  );
};

export default RectangularSelectionTool;
