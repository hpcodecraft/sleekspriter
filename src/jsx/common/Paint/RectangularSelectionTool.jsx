var RectangularSelectionTool = React.createClass({
  render: function() {
    return (
      <div id="RectangularSelection-Tool" className="ToolComponent">
        <i className="icon flaticon-selection7"></i>
        <span className="hint">Select some pixels to work with!</span>
      </div>
    );
  }
});