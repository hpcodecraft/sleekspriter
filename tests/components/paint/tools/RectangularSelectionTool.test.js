import React from "react";
import RectangularSelectionTool from "components/paint/tools/RectangularSelectionTool";

describe("RectangularSelectionTool", () => {
  it("should render correctly", () => {
    const component = shallow(<RectangularSelectionTool />);
    expect(component.html()).toMatchSnapshot();
  });
});
