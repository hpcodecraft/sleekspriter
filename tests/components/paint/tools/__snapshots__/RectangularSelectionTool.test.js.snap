// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RectangularSelectionTool should render correctly 1`] = `"<div id=\\"RectangularSelection-Tool\\" class=\\"ToolComponent\\"><i class=\\"icon flaticon-selection7\\"></i><span class=\\"hint\\">Select some pixels to work with!</span></div>"`;
